src/dmd/frontend.h

```diff
@@ -5031,21 +5031,18 @@ struct TargetC final
         Gcc_Clang = 3u,
     };
 
-    bool crtDestructorsSupported;
```

Comment:
what is this?

Reply:
My bad. I forgot to regenerate frontend.h after rebasing.